sys/include/net/ng_netconf.h

```diff
@@ -119,8 +119,12 @@ typedef enum {
      * @note not all transceivers may support this interrupt
      */
     NETCONF_OPT_TX_END_IRQ,
-    NETCONF_OPT_AUTOCCA,            /**< en/disable to check automatically
-                                         before sending the channel is clear. */
+    NETCONF_OPT_CSMA,               /**< en/disable to check automatically
```

Comment:
> Doesn't this value back off the time for sensing the channel? If the channel is sensed clear, the packet should always be sent immediately, right?

Nope ;) Have a look at the 802.15.4 specification. See page 172 of the 2006 version (Figure 69) for the actual CSMA/CA algorithm. Before every channel sensing, there's the random backoff waiting.